sds_in_a_box/SDSCode/upload_api.py

```diff
@@ -0,0 +1,117 @@
+import json
+import os
+import requests
+import time
+import boto3
+from jose import jwk, jwt
+from jose.utils import base64url_decode
```

Comment:
should we remove these? They are not used

Reply:
Ah yeah sorry, these imports are actually a part of the cognito pull request (which I'm saving for last, after all these things are properly reviewed).  These will be added in again for that pull request